.github/workflows/ci.yaml

```diff
@@ -95,6 +95,6 @@ jobs:
       - name: Test coverage
         if: success() && runner.os == 'Linux' && matrix.config.r == 'release'
         run: |
-          remotes::install_cran("covr")
+          remotes::install_cran("covr", INSTALL_opts = "${{ matrix.config.multiarch }}")
```

Comment:
did you add it here intentionally as well?

Reply:
Yes, to be on the safe side... the problem is with installing from source (i.e. not from the .zip binary) any package with compiled code, which cannot be installed for both architecture if any of its dependencies were not installed for both architecture (which is the case for packages installed from binaries on r-devel Windows).

Instead of checking whether this could be the case (with no guarantee it will stay the same in the future) for specific packages like `covr` or `rcmdcheck`, I went with the approach of making sure all workflow steps would run with `--no-multiarch`, which applies to `R CMD check` and package installs (in any of the `install*` forms we use).